src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -71,8 +75,8 @@ private LimelightData(double valid, double xAngle, double yAngle) {
       this.xCoord = xAngle;
       this.yCoord = yAngle;
 
-      double height = TARGET_HEIGHT - CAMERA_HEIGHT;
-      double slope = Math.tan(CAMERA_ANGLE + yAngle);
+      double height = Constants.TARGET_HEIGHT - Constants.CAMERA_HEIGHT;
+      double slope = Math.tan(Math.toRadians(Constants.CAMERA_ANGLE) + Math.toRadians(yAngle));
```

Comment:
Instead of calling toRadians twice, add the angles as degrees and then convert to radians once.